src/main.ts

```diff
@@ -0,0 +1,30 @@
+import * as core from '@actions/core'
+import {GitHubClient} from './github'
+
+/* When an issue or pull request is opened we want to look at the associated user and
+see if they are a member of the X organisations and Y teams. If they are not we should
+label the issue with the label `community`. */
+async function run(): Promise<void> {
+  try {
+    // Retrieve our inputs
+    const labelName = core.getInput('label_name', {required: true})
+    const labelColor = core.getInput('label_color', {required: true})
+    const loginsToIgnore = core.getInput('logins_to_ignore', {required: false})
+    const orgs = core.getInput('orgs', {required: true}).split(',')
```

Comment:
Should this be `org_membership` instead of `orgs`?

Reply:
I've resolved these two comments & added a test for the scenario.

Thanks!